locales/en-US/app.ftl

```diff
@@ -0,0 +1,125 @@
+landingDownloadFirefoxSubTitle = Free Download
+landingExperimentsTitle = Test these features and share your feedback
+landingCardListTitle = Get started in 3 easy steps
+landingStepOne = Step one
+landingStepTwo = Step two
+landingStepThree = Step three
+landingCardOne = Create a Firefox Account or sign in.
+landingCardTwo = Get the Test Pilot add-on.
+landingCardThree = Enable experimental features!
+
+errorHeading = Whoops!
+errorMessage = Looks like we broke something. <br> Maybe try again later.
+
+notFoundHeader = Four Oh Four!
+
+experimentListPageHeader = Ready for Takeoff!
+experimentListPageSubHeader  = Pick the features you want to try. <br> Check back soon for more experiments.
+
+experimentListInactiveHover = Get Started
+experimentListActiveHover = Manage
+
+giveFeedback = Give Feedback
+
+disableExperiment = Disable {$title}
+disableExperimentTransition = Disabling...
+enableExperiment = Enable {$title}
+enableExperimentTransition = Enabling...
+
+feedbackSubmitButton = Take a quick survey
+feedbackCancelButton = Close
+
+feedbackUninstallTitle = Thank You!
+feedbackUninstallCopy =
+    | Your participation in Firefox Test Pilot means
+    | a lot! Please check out our other experiments,
+    | and stay tuned for more to come!
+
+discussNotifyTitle = Just one second...
+discussNotifyMessage =
+    | <p>In the spirit of experimentation, we're using a forum service
+    | that requires an account different from the one you
+    | created for Test Pilot. We are working hard to change this
+    | and make the forum experience smoother.</p>
+    | <p>If you don't want to create another account, you can
+    | always leave feedback through Test Pilot.
+    | <br>
+    | (We really do read this stuff)</p>
+
+discussNotifySubmitButton = Take me to the forum
+discussNotifyCancelButton = Cancel
+
+retireDialogTitle = Uninstall Test Pilot?
+retireMessage = As you wish. This will disable any active tests, uninstall the add-on and remove your account info from our servers.
+retireSubmitButton = Proceed
+retireCancelButton = Cancel
+
+pageTitleRetirePage = Firefox Test Pilot - Uninstall Test Pilot
+retirePageProgressMessage = Shutting down...
+retirePageHeadline = Thanks for flying!
+retirePageMessage = Hope you had fun experimenting with us. <br> Come back any time.
+retirePageSurveyButton = Take a quick survey
+
+measurements = Your privacy
+experimentPrivacyNotice = You can learn more about the data collection for {$title} here.
+contributorsHeading = Brought to you by
+version = Version
+changelog = changelog
+lastUpdate = Last Update
+contribute = Contribute
+bugReports = Bug Reports
+discourse = Discourse
+nowActive = Active
+userCount =
+    [html/title] Active Users of this Experiment
+tourOnboardingTitle = {$title} enabled!
+tourStartButton = Take the Tour
+tourCancelButton = Skip
+tourDoneButton = Done
+userCountContainer = There are <span class="bold">{$installation_count}</span> people trying {$title} right now!
```

Comment:
The `class` attribute will be filtered out by l20n.js so it can be removed here. It should be defined on the source `<span>` in the HTML—l20n.js will reuse that original `<span>` and the `class` will be preserved.